example/pubspec.lock

```diff
@@ -149,6 +191,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.8"
+  xml:
+    dependency: transitive
+    description:
+      name: xml
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "3.5.0"
 sdks:
-  dart: ">=2.2.2 <3.0.0"
+  dart: ">=2.4.0 <3.0.0"
```

Comment:
I'm not sure we want to bump this min SDK. This would be a breaking change. Are there features in 2.2.2+ you require for this PR?

Reply:
The bump here is a mistake on my part, the change in min sdk is not required.